app/controllers/api/v1/applications_controller.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
the confusing thing to me is why it isn't causing rubocop to fail when we run it on semaphore, because when I run rubocommit locally it gets made about all the files, not just the files i've touched.